test/extensions/common/wasm/wasm_vm_test.cc

```diff
@@ -258,14 +257,14 @@ TEST_P(WasmVmTest, V8Memory) {
   ASSERT_TRUE(wasm_vm != nullptr);
 
   auto code = TestEnvironment::readFileToStringForTest(TestEnvironment::substitute(
-      "{{ test_rundir }}/test/extensions/common/wasm/test_data/test_rust.wasm"));
+      "{{ test_rundir }}/test/extensions/common/wasm/test_data/wee8_test_rust.wasm"));
   EXPECT_TRUE(wasm_vm->load(code, GetParam()));
 
   wasm_vm->registerCallback("env", "pong", &pong, CONVERT_FUNCTION_WORD_TO_UINT32(pong));
   wasm_vm->registerCallback("env", "random", &random, CONVERT_FUNCTION_WORD_TO_UINT32(random));
   wasm_vm->link("test");
 
-  EXPECT_EQ(wasm_vm->getMemorySize(), 65536 /* stack size requested at the build-time */);
+  EXPECT_EQ(wasm_vm->getMemorySize(), 1114112 /* stack size requested at the build-time */);
```

Comment:
I don't know to configure stack-size on LLVM via rules_rust. So I changed to follow.

Reply:
I think you could pass it to `rustc` via `rustc_flags` using `-C link-arg=-zstack-size=32768`.